.github/CODEOWNERS

```diff
@@ -12,3 +12,9 @@ charts/ @bitwarden/dept-shot
 
 ## BRE team owns these workflows ##
 .github/workflows/release.yml @bitwarden/dept-bre
+
+# Docker-related files
+**/Dockerfile @bitwarden/team-appsec @bitwarden/dept-bre
+**/*.dockerignore @bitwarden/team-appsec @bitwarden/dept-bre
+**/entrypoint.sh @bitwarden/team-appsec @bitwarden/dept-bre
+**/docker-compose.yml @bitwarden/team-appsec @bitwarden/dept-bre
```

Comment:
`@bitwarden/dept-shot` here as well, similar to https://github.com/bitwarden/unified-environments/commit/ef626decd10157ac1417b0902a0e7faaeb0e8e9f